extensions/account-registration.md

```diff
@@ -0,0 +1,204 @@
+---
+title: `account-registration` Extension
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Ed Kellett"
+    period: "2020"
+    email: "e@kellett.im"
+  -
+    name: "Val Lorentz"
+    period: "2021"
+    email: "progval+ircv3@progval.net"
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use
+the unprefixed `account-registration` capability name.
+Instead, implementations SHOULD use the `draft/account-registration`
+capability name to be interoperable with other software implementing
+a compatible work-in-progress version.
+
+The final version of the specification will use an unprefixed capability name.
+
+## Motivation
+
+The lack of a standardized way to register an account has been a source
+of frustration for many participants for several years.
+
+This specification is intentionally restricted in scope to be appealing
+to implementors by being severely. The authors hope that this solution
+will serve as the basis for future organic expansion.
```

Comment:
How about this? Historically, other specs lean on the word "historically" in this section.

```suggestion
A standard method of registering a user account has historically been missing
from IRC. Instead various methods such as sending messages to services bots
(e.g. NickServ) have been employed.

This specification aims to provide a standard baseline for account
registration that allows client developers to build more reliable and
integrated user experiences around account registration.

It does not aim to cover all registration-related use cases that may currently
be in use by server implementations to varying degrees. However, future
extensions to the specification might be proposed given a broad enough
consensus.
```

Reply:
Nice!